code/assurance/src/main/java/com/adobe/marketing/mobile/assurance/internal/ui/AssuranceActivity.kt

```diff
@@ -46,49 +42,23 @@ class AssuranceActivity : ComponentActivity() {
                             .fillMaxSize(),
                         color = backgroundColor
                     ) {
-                        Box(
```

Comment:
Can you check how this looks with edge-to-edge enabled?  We need this `Box` with a safe drawing padding to ensure that the text does not get cut off in a phone use case.

Reply:
I will test on edge-to-edge.  But I think I accidentally removed these code, I put it back now.